articles/azure-sql/migration-guides/virtual-machines/db2-to-sql-on-azure-vm-guide.md

```diff
@@ -143,8 +143,8 @@ For additional assistance, see the following resources, which were developed in
 |Asset  |Description  |
 |---------|---------|
 |[Data workload assessment model and tool](https://github.com/Microsoft/DataMigrationTeam/tree/master/Data%20Workload%20Assessment%20Model%20and%20Tool)| This tool provides suggested "best fit" target platforms, cloud readiness, and application/database remediation level for a given workload. It offers simple, one-click calculation and report generation that helps to accelerate large estate assessments by providing and automated and uniform target platform decision process.|
-|[Db2 zOS data assets discovery and assessment package](https://github.com/microsoft/DataMigrationTeam/tree/master/DB2%20zOS%20Data%20Assets%20Discovery%20and%20Assessment%20Package)|After running the SQL script on a database, you can export the results to a file on the file system. Several file formats are supported, including *.csv, so that you can capture the results in external tools such as spreadsheets. This method can be useful if you want to easily share results with teams that do not have the workbench installed.|
-|[IBM Db2 LUW inventory scripts and artifacts](https://github.com/microsoft/DataMigrationTeam/tree/master/IBM%20DB2%20LUW%20Inventory%20Scripts%20and%20Artifacts)|This asset includes a SQL query that hits IBM Db2 LUW version 11.1 system tables and provides a count of objects by schema and object type, a rough estimate of "raw data" in each schema, and the sizing of tables in each schema, with results stored in a CSV format.|
+|[Db2 zOS data assets discovery and assessment package](https://aka.ms/Script-DB2zOSFeatureAssessmentPackage)|After running the SQL script on a database, you can export the results to a file on the file system. Several file formats are supported, including *.csv, so that you can capture the results in external tools such as spreadsheets. This method can be useful if you want to easily share results with teams that do not have the workbench installed.|
```

Comment:
Feedback from governance. Please update to a direct link. These links are not recommended for the following reasons:
-	It’s bad for SEO.
-	If it points to another Doc, it won’t work in Air-Gapped Clouds (ShortLink services aren’t present).
-	Tooling like the OPS Broken Links report likely won’t tell you it’s broken because the ShortLink itself doesn’t return a 404.
-	It’s hidden from the text, so it’s not really making it a “short link” for the purposes of sharing.  It’s just another way to hide content debt.

It may be a blocking issue for PR Reviewers because it is not recommended. More direct language is being considered for the Contributor Guide for Content.